docs/shell.md

```diff
@@ -40,17 +40,14 @@ $ inspec shell -t winrm://UserName:Password@windowsmachine:1234  # Login to wind
 $ inspec shell -t docker://container_id # Login to a docker container.
 ```
 
-## Resource packs
+## Resource Packs
 
-The InSpec shell may use additional keywords provided in resource packs.
-A resource pack is a profile that defines new language terms that can
-be used in InSpec. For example, the profile in `examples/profile` in
-the InSpec git repo defines a `gordon_config` resource. To use these
-resources with the InSpec shell, you will need to download and specify
-them as a dependency.
+Use resource packs to share custom resources with other InSpec users.
+A resource pack is an inspec profile that contains only custom resources and no other controls or tests. 
 
-To use the `gordon_config` resource that is provided in the `examples/profile`
-in the InSpec repo you can run the following:
+For example, the profile in [`examples/profile`](https://github.com/chef/inspec/tree/master/examples/profile)in the InSpec git repo defines a [`gordon_config` resource](https://github.com/chef/inspec/blob/master/examples/profile/controls/gordon.rb). To use these resources within the InSpec shell, you will need to download and specify them as a dependency.
```

Comment:
Do we have this documented anywhere?

> you will need to download and specify them as a dependency.

Reply:
In the profile & profile inheritance section